### PR TITLE
Share a launch via text

### PR DESCRIPTION
**Pull request by Celinarabe**
Created the method `shareLaunch()` within the launch details fragment. The method creates a text message with launch information. If any field is null, it will be omitted from the text message.

https://user-images.githubusercontent.com/41392379/166403320-dcbd479e-e09c-4f43-b258-020aed71c361.mov

### CODE REVIEW

File: app/src/main/java/com/example/voyagerx/LaunchDetailsFragment.kt [hunk 1]
```diff
@@ -116,4 +117,17 @@ class LaunchDetailsFragment : Fragment() {
     private fun hideView(view: View) {
         view.visibility = View.GONE
     }
+
+    private fun shareLaunch() {
+        val share = Intent.createChooser(Intent().apply {
```
Comment (chrsbell) [inline app/src/main/java/com/example/voyagerx/LaunchDetailsFragment.kt:122]: I was thinking the chooser would be more complicated to do, this is pretty neat

**Review by chrsbell — APPROVED** [review]
Looks good!

File: app/src/main/java/com/example/voyagerx/LaunchDetailsFragment.kt (hunk 1 above)
Comment (Celinarabe) [reply app/src/main/java/com/example/voyagerx/LaunchDetailsFragment.kt:122]: I know right! Super easy

**Review by VivianLi98 — APPROVED** [review]
Wooo! Looks good

**Pull request merged**